### PR TITLE
remove duplicated code

### PR DESCRIPTION
**Pull request by liubin**
Lines 138-140 are the same as 134-136

### CODE REVIEW

**Review by milosgajdos — APPROVED** [review]
Nice catch. Thanks!

File: registry/storage/paths.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ const (
 //	blobsPathSpec:                  <root>/v2/blobs/
 //	blobPathSpec:                   <root>/v2/blobs/<algorithm>/<first two hex bytes of digest>/<hex digest>
 //	blobDataPathSpec:               <root>/v2/blobs/<algorithm>/<first two hex bytes of digest>/<hex digest>/data
-//	blobMediaTypePathSpec:               <root>/v2/blobs/<algorithm>/<first two hex bytes of digest>/<hex digest>/data
```
Comment (milosgajdos) [inline registry/storage/paths.go:111]: can you move this to a separate PR please

**Review by milosgajdos — CHANGES_REQUESTED** [review]

**Review by Jamstah — APPROVED** [review]

File: registry/storage/paths.go (hunk 1 above)
Comment (liubin) [reply registry/storage/paths.go]: @milosgajdos updated.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by milosgajdos — APPROVED** [review]
